science/orthanc-mysql/pkg-plist

```diff
@@ -0,0 +1,4 @@
+share/orthanc/plugins/libOrthancMySQLIndex.so
+share/orthanc/plugins/libOrthancMySQLIndex.so.4.0
```

Comment:
`4.0` can be replaced by something in PLIST_SUB and so as other orthanc plugin ports.